src/consts/models.tsx

```diff
@@ -121,6 +121,8 @@ export enum Route {
   ChunkedQrCode = 'ChunkedQrCode',
 }
 
+export type WalletType = '3-Key Vault' | '2-Key Vault' | 'Standard HD P2SH' | 'Standard P2SH' | 'Standard HD SegWit';
```

Comment:
The whole WalletType should be built from merging all wallets class types

Reply:
I'm afraid it's not possible. I implemented this type only to have a strongly typed identifier for mapping purposes. Doing it this way
```
export type WalletType = HDSegwitP2SHAirWallet | HDSegwitP2SHArWallet | (...)
```

gives me nothing - I have no idea how could I pass it as a value into `RadioButton`. If our wallets would be implemented in TS I'd propose something like:
```
type WalletType = 'Foo Wallet' | 'Bar Wallet' | (...)

class FooWallet {
  type: WalletType = 'Foo Wallet';
}

(...)
```

but they're not. 

Does it make sense? If I missed something let's have a call

Reply:
Also, it's not possible to build a type using variables. So, a type implemented like this:
```
export type WalletType = [HDSegwitP2SHAirWallet.type] | [HDSegwitP2SHArWallet.type] | (...)
```

is not allowed

Reply:
indeed the computed value is not allowed in type, my bad